examples/Workflow/README.md

```diff
@@ -25,6 +25,20 @@ dapr run --app-id wfwebapp dotnet run
 
 The application will listen for HTTP requests at `http://localhost:10080`.
 
+This workflow example utilizes a redis statestore. In order to retrieve items from this statestore, an HTTP command must first be sent down to restock the inventory:
+
+On Linux/macOS (bash):
+
+```bash
+curl -i -X POST http://localhost:10080/reset
+```
+
+On Windows (PowerShell):
+
+```powershell
+curl -i -X POST http://localhost:10080/reset `
```

Comment:
This command actually isn't correct for PowerShell. The trailing ` means you're entering a multi-line command, which is not the case.

In any case, the command is exactly the same for both bash and powershell, so no need to distinguish them here.